releases/0.3.0.heads.md

```diff
@@ -0,0 +1,134 @@
+# Dasharo Trustworthy Computing
+
+[https://dasharo.com](https://dasharo.com)
+
+Talos II support in Heads.
+
+## v0.3.0 Heads
+
+This release contains changes only for Heads.
+There are no coreboot changes included in this release.
+
+### Release Changelog
+\
+**Added:**
+* Support for PPC64 architecture - first non-x86 target for Heads.
+* Improve project structure introducing subdirectory per architecture.
+* Replace some hard-coded paths with variables.
+
+### Upstream status
+
+#### Pending
+
+Lines added: 14843
+Lines removed: 25
+
+#### Active
+
+* [55038: src/lib/fmap.c: use le*toh() functions where needed](https://review.coreboot.org/c/coreboot/+/55038)
+* [55039: util/cbfstool/flashmap/fmap.c: fix fmaptool endianness bugs on BE](https://review.coreboot.org/c/coreboot/+/55039)
+
+Lines added: 49
+Lines removed: 38
+
+#### Merged
+
+* [55037: ppc64/byteorder.h: define use of big endian](https://review.coreboot.org/c/coreboot/+/55037)
+* [55040: commonlib/lz4_wrapper.c: do not use bitfields](https://review.coreboot.org/c/coreboot/+/55040)
+* [55041: docs/flashmap: state the endianness of FMAP](https://review.coreboot.org/c/coreboot/+/55041/3)
+
+Lines added: 36
+Lines removed: 41
+
+### Download firmware images
+
+#### Release binaries
+
+* [dasharo-trustworthy-computing-v0.3.0.rom]()
```

Comment:
@IgorBagnucki We will not publish new coreboot binaries, only heads

Reply:
Should there be only one binary and one checksum or something like that? I'm not sure how it affects the links.

Reply:
Yes, the `zImage.bundled` will be the only one released binary it seems. Each released binary also have to accompanying files: .SHA256 and .SHA256.sig

Reply:
@macpijan I have removed most links, leaving only one for binary and one for checksum.